scripts/run-swift-format.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftASN1 open source project
+##
+## Copyright (c) 2022-2023 Apple Inc. and the SwiftASN1 project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftASN1 project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftOpenAPIGenerator open source project
+##
+## Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+set -euo pipefail
+
+log() { printf -- "** %s\n" "$*" >&2; }
+error() { printf -- "** ERROR: %s\n" "$*" >&2; }
+fatal() { error "$@"; exit 1; }
+
+CURRENT_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_ROOT="$(git -C "${CURRENT_SCRIPT_DIR}" rev-parse --show-toplevel)"
+
+SWIFTFORMAT_BIN=${SWIFTFORMAT_BIN:-$(command -v swift-format)} || fatal "❌ SWIFTFORMAT_BIN unset and no swift-format on PATH"
+
+"${SWIFTFORMAT_BIN}" lint \
+  --parallel --recursive --strict \
+  "${REPO_ROOT}/Sources" "${REPO_ROOT}/Tests" \
+  && SWIFT_FORMAT_RC=$? || SWIFT_FORMAT_RC=$?
+
+if [ "${SWIFT_FORMAT_RC}" -ne 0 ]; then
```

Comment:
```suggestion
if [[ "${SWIFT_FORMAT_RC}" -ne 0 ]]; then
```

Reply:
I know that we explicitly call out bash in the shebang, but the single-brackets are more portable, POSIX.